Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/Definitions/Result.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON.Definitions
+{
+    public class Result
+    {
+        /// <summary>
+        /// Column name. If input data contains Header row and value is left empty, header value is used as name.
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Text")]
+        public string Data { get; set; }
```

Comment:
You should use `private set` here and you need to add a constructor to the class. This is because we have decided that after an object is created of the result class, you should not be able to modify the data of that object.